app/models/course.rb

```diff
@@ -265,6 +265,10 @@ def average_word_count
     word_count / user_count
   end
 
+  def last_update
```

Comment:
This is not specific to a course, so it shouldn't be a method on course. MetricsUpdate can provide this directly.